### PR TITLE
Cli import csv feature

### PR DESCRIPTION
**Pull request by harobed**
https://github.com/modoboa/modoboa-admin/issues/15

### CODE REVIEW

**Comment by tonioo** [comment]
Few things:
* It looks like this PR also includes the first one (rename import.py)
* Instead of duplicating the code used to load the proper import function, could you please extract it into a dedicated module and use it inside the CLI command and inside the view ?

**Comment by harobed** [comment]
> Instead of duplicating the code used to load the proper import function, could you please extract it into a dedicated module and use it inside the CLI command and inside the view ?

Like my last commit ?

**Comment by tonioo** [comment]
Yes, but now unit tests fail :)

File: modoboa_admin/management/commands/_import.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            try:
+                fct = globals()["import_%s" % row[0].strip()]
+            except KeyError:
+                fct = events.raiseQueryEvent(
+                    'ImportObject', row[0].strip())
+                if not fct:
+                    continue
+                fct = fct[0]
+
+            print(row)
```
Comment (tonioo) [inline modoboa_admin/management/commands/_import.py]: Debug ?

**Comment by harobed** [comment]
How can I reproduce this test environment in my Debian Vagrant host ?
I've postgresql installed, I execute : 

```
(vtenv)vagrant@modoboa:~/modoboa-admin$ python -m unittest -v -f tests
test (tests.TestRunner)
Run extensions tests. ... Creating test database for alias 'default'...
E..........EEEE.................F....EEEEEEE..E...
======================================================================
ERROR: test_crud (modoboa_admin.tests.account.AccountTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
```

**Comment by harobed** [comment]
Ok, I've the same test error : 

```
(vtenv)vagrant@modoboa:~/modoboa_instance$ python manage.py  test modoboa_admin.tests
Creating test database for alias 'default'...
................................F.................
======================================================================
FAIL: test_import_quota_too_big (modoboa_admin.tests.import_.ImportTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/modoboa-admin/modoboa_admin/tests/import_.py", line 127, in test_import_quota_too_big
    self.assertIn('Quota is greater than the allowed', resp.content)
AssertionError: 'Quota is greater than the allowed' not found in '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"\n "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">\n<html xmlns="http://www.w3.org/1999/xhtml">\n<head profile="http://www.w3.org/2005/10/profile">\n  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />\n  <script type="text/javascript">\n    parent.admin.importdone("ko", "Permission denied: You can\\u0027t import an account with a role greater than yours");\n  </script>\n</head>\n<body></body>\n</html>\n'

----------------------------------------------------------------------
Ran 50 tests in 35.775s

FAILED (failures=1)
Destroying test database for alias 'default'...
```

**Pull request merged**